components/DescriptionList/src/index.css

```diff
@@ -0,0 +1,61 @@
+.denhaag-descriptionlist {
+  color: var(--denhaag-descriptionlist-color, inherit);
+  display: grid;
+  grid-template-columns: repeat(1, 1fr);
+  font-family: var(--denhaag-descriptionlist-font-family, inherit);
+  font-size: var(--denhaag-descriptionlist-font-size, inherit);
+  font-weight: var(--denhaag-descriptionlist-font-weight, inherit);
+  line-height: var(--denhaag-descriptionlist-line-height, inherit);
```

Comment:
These properties are already inherited if their tokens are undefined. I believe we can remove the `inherit` fallback. (Didn't test on safari). Occurs more often further down in the css file.